README.md

```diff
@@ -1,30 +1,5 @@
 # Models
 
-## Meal Request Object
-This will be sent from the front end to the back end to request the kind of meal desired.
+## Meal Request Objects
+**To see what the objects look like [here](src/kotlin/edu/byu/mealplanningassistant/models/Recipe.kt)**
```

Comment:
Note. The link won't work until we merge the PR as that file does not yet exist in the repo.